build.gradle

```diff
@@ -25,6 +25,8 @@ dependencies {
     implementation  group: 'org.json', name: 'json', version: '20180130'
 
     testImplementation group: 'junit', name: 'junit', version: '4.12'
+
+    compile group: 'org.kohsuke', name: 'github-api', version: '1.93'
```

Comment:
compile is sort of a deprecated function, use  implementation instead and also group together with the other implementation.